batch encoder.bat

```diff
@@ -32,9 +32,10 @@ if /i "%1"=="--updated-from" (
 	curl -L -H "Accept: application/vnd.github+json" -o batch_update.json https://api.github.com/repos/Adam-Kay/Batch-Encoder/releases/latest
 	rem curl --silent -o batch_update.txt https://gist.githubusercontent.com/Adam-Kay/ec5da0ff40e8eb14beee2242161f5191/raw
 	
-	>%TEMP%\foo.tmp findstr "prerelease" batch_update.json
-	<%TEMP%\foo.tmp set /p "prerelease="
-	echo %prerelease%
+	>%TEMP%\batch_update.tmp findstr "prerelease" batch_update.json
+	<%TEMP%\batch_update.tmp set /p "prerelease_entry="
+	set "prerelease=%prerelease_entry:~16,-1%"
+	echo prerelease? : %prerelease%
```

Comment:
This will _always_ return `prerelease? : false` because [line 32](https://github.com/Adam-Kay/Batch-Encoder/blob/362f2c1/batch%20encoder.bat#L32) connects to `/releases/latest`, and it is not possible for a release to be a **latest** and a **prerelease** simultaneously. Ergo, this is likely not needed.